mbed-os.lib

```diff
@@ -1 +1 @@
-https://github.com/ARMmbed/mbed-os/#0db72d0cf26539016efbe38f80d6f2cb7a3d4414
+https://github.com/ARMmbed/mbed-os/
```

Comment:
Un-versioned mbed-os.lib is not compatible with the current release script yet, if I remember correctly.

For now, just rebase this PR to the latest development branch so `mbed-os.lib` points to the latest 6.5.0 release which is new enough.

Reply:
Okay, didn't know the script is updated.